doc/code/sf_cli.rst

```diff
@@ -8,50 +8,56 @@ including:
 
 * ``sf configure [--token] [--local]``: configure the connection to the cloud platform
 
-* ``sf run input [--output FILE]``: submit and execute quantum programs from the command line
+* ``sf run INFILE [--output FILE]``: submit and execute quantum programs from the command line

```

Comment:
according to the PR, shouldn't this be clarified to be a blackbird script?

Reply:
Oh weird, I thought I had done this 🤔 
```suggestion
* ``sf run INFILE [--output FILE]``: submit and execute Blackbird scripts containing quantum programs from the command line
```